CRM/Core/BAO/CustomField.php

```diff
@@ -1217,7 +1217,12 @@ private static function formatDisplayValue($value, $field, $entityId = NULL) {
       case 'Multi-Select State/Province':
       case 'Multi-Select Country':
         if ($field['data_type'] == 'ContactReference' && $value) {
-          $display = CRM_Core_DAO::getFieldValue('CRM_Contact_DAO_Contact', $value, 'display_name');
+          if (is_int($value)) {
```

Comment:
I feel like is_numeric would be safer. my understanding is that if it were '1' then that would be FALSE in is_int's world. We are not very good at our casting...

Reply:
Ok, changed to `is_numeric`